src/pages/RadicalCollaboration/index.js

```diff
@@ -83,12 +30,95 @@ const RadicalCollaboration = () => {
       </Box>
       <Box className='containerGray'>
         <Container>
-          <Hidden mdDown>
-            <PhotoCardDesktop items={Images} />
-          </Hidden>
-          <Hidden lgUp>
-            <PhotoCardMobile items={Images} />
-          </Hidden>
+          <Grid container item>
+            <Grid>
+              <Paper elevation={0} className={classes.innerTextCardContainer}>
+                <Typography className={classes.cardHeading} variant='h4'>Donate </Typography>
+                <Typography variant="h6" color='primary' className={classes.subcardHeading}>Every gift helps us continue our work.</Typography>
+                <Typography className={classes.dtpLine1}>
+                 Your tax-deductible gift today will help make more open-source solutions easily available for the
+                 communities who need it the most. Thank You
+                </Typography>
+                <Button component={Link} to="/donate"  color='textSecondary' className={classes.donateButton}>
+                 Make a Donation
+                </Button>
+                <br></br>
+              </Paper>
+            </Grid>
+            <Grid>
+              <Card className={classes.imgCard}>
+                <CardMedia
+                  className={classes.cardMedia}
+                  image={'/images/Donate-to-Us.png'}
+                />
+              </Card>
+            </Grid>
+            <Grid>
+              <Paper elevation={0} className={classes.innerLeftTextCardContainer}>
+                <Typography className={classes.shareCardHeading} variant='h4'>Share the CTI </Typography>
+                <Typography variant="h6" color='primary' className={classes.subcardHeading}>Love the Index? Be an evangelist!</Typography>
+                <Typography className={classes.rightDtpLine1}>
+                 Help others and their communities discover, share and benefit from contributed projects on the index.
+                </Typography>
+                <Button component={Link} to='/support/share'  className={classes.buttonRight}>
+                 Share the CTI
+                </Button>
+                <br></br>
```

Comment:
What are these line breaks doing? Are they a divider between each block of text & image?

Reply:
I removed those, Thanks Dennis